docs/design/proposals/encryption-with-vault-tokens.md

```diff
@@ -0,0 +1,125 @@
+# Multi-tenancy with Vault Tokens
+
+## Current Feature: Vault access with single token
+
+The current feature to [support Hashicorp Vault as a KMS is
+documented](./encrypted-pvc.md):
+
+- tenants can create PVCs that are encrypted with a unique key
+- the key to encrypt/decrypt the PVC is stored in Hashicorp Vault
+- there is a single token (Kubernetes ServiceAccount) to access Vault
```

Comment:
What about the Vault token access using the token auth method and not the Kubernetes Auth?

Reply:
That is what we're adding now. This part of the **Current Feature** is about what is supported now, not what is going to be added.

Reply:
Ok, thanks. Let's just replace "token" with "JSON Web Token" (JWT) for more accuracy.
"